### ISSUE TITLE
Remove non-existent HP xrefs (1)

### ISSUE THREAD

**Issue by asmvernon** [monarch-initiative/mondo#1176]
**Mondo term (ID and Label):**
MONDO:0009576

**Xref that should be fixed (ID and label):**
HP:0007660 should be removed as it is not a valid HP ID

**Other comments:**
nonexistent

**Comment by nicolevasilevsky** [comment]
this is in HPO:
https://hpo.jax.org/app/browse/term/HP:0000485
